src/test/java/com/puppycrawl/tools/checkstyle/gui/ParseTreeTablePModelTest.java

```diff
@@ -163,8 +163,109 @@ public void testJavadocChild() {
         Assert.assertTrue(javadoc instanceof DetailNode);
         Assert.assertEquals(JavadocTokenTypes.JAVADOC, ((DetailNode) javadoc).getType());
         final Object javadocChild = parseTree.getChild(javadoc, 2);
-        Assert.assertTrue(javadoc instanceof DetailNode);
+        Assert.assertTrue(javadocChild instanceof DetailNode);
         Assert.assertEquals(JavadocTokenTypes.TEXT, ((DetailNode) javadocChild).getType());
     }
 
+    @Test
+    public void testGetIndexOfChild() {
+        final DetailAST nChild = tree.getFirstChild().getNextSibling().getNextSibling();
+        final ParseTreeTablePModel parseTree = new ParseTreeTablePModel(null);
+        final int index = parseTree.getIndexOfChild(tree, nChild);
+        Assert.assertEquals(2, index);
```

Comment:
Just confirming, 2 is because we called `getNextSibling()` twice? If so, maybe add a comment to note that.